test/test_coordinates.py

```diff
@@ -24,6 +24,10 @@ def test_basic(self):
         self.assertEqual(coord.first, coord.x)
         self.assertEqual(coord.second, coord.y)
         self.assertEqual(coord, coord.copy())
+        self.assertEqual(str(coord), "(1, 2)")
+        self.assertEqual(hash(coord), 3713081631934410656)
```

Comment:
It's a while since I wrote these (I forgot to PR them), but I guess it was to cover the [`__hash__` function](https://github.com/sethoscope/heatmap/blob/9e1f27b6a46b5ea02969209616fd2137eff2942e/heatmap.py#L55-L56).

We could make it more general, and still be able to cover it so we know it's runnable and returns something sensible for all Python versions. Perhaps just check the return is an int? Or just that it's callable with no exception?